apps/fabric-website/src/components/App/App.scss

```diff
@@ -40,10 +33,17 @@
     transition: all $ms-animation-duration-2 $ms-animation-ease-1;
 
     @include high-contrast {
-      border-right: 1px solid WindowText;
-      border-left: 1px solid WindowText;
-      border-bottom: 1px solid WindowText;
+      border-right: 1px solid $ms-color-white;
+      border-left: 1px solid $ms-color-white;
+      border-bottom: 1px solid $ms-color-white;
+    }
+
+    @include high-contrast-black-on-white  {
+      border-right: 1px solid $ms-color-black;
+      border-left: 1px solid $ms-color-black;
+      border-bottom: 1px solid $ms-color-black;
```

Comment:
Why did this get changed back from using the `WindowText` system color? Using the system color allows it to support any high-contrst palette with a single media query.

Reply:
This is likely due to the crazy branch switching we've been doing - I'll look into it!

Reply:
Yeah, @kristoferbrown, I think it was the former - I'll switch it back. Thanks for paying attention!